src/core/core.tooltip.js

```diff
@@ -532,14 +542,15 @@ module.exports = function(Chart) {
 				tooltipSize = getTooltipSize(this, model);
 				alignment = determineAlignment(this, tooltipSize);
 				// Final Size and Position
-				backgroundPoint = getBackgroundPoint(model, tooltipSize, alignment);
+				backgroundPoint = getBackgroundPoint(model, tooltipSize, alignment, me._chart);
 			} else {
 				model.opacity = 0;
 			}
 
 			model.xAlign = alignment.xAlign;
 			model.yAlign = alignment.yAlign;
 			model.x = backgroundPoint.x;
+			model.xCaret = backgroundPoint.xCaret;
```

Comment:
Would it make sense / be useful to also have a way to override the `yCaret` position as well?